### DISCUSSION TITLE
Will this work with Docker Compose v2.2 and above?

### DISCUSSION THREAD

**Discussion by vmcosco** [tiredofit/docker-traefik-cloudflare-companion#65] [Q&A] [answered]
There is a fairly popular and comprehensive Traefik/Clouflare/Home Server guide out there that indicates that this image does not seem to work with Docker compose versions 2.2 or higher.  Is this still (or was it ever) the case?  I'm interested in using this but am already running Docker Compose v1.29.2.  Thanks

**Comment by tiredofit** [comment, marked as answer]
There shouldn't be any problem running with Docker Compose 2.x as it is simply just a front end or orchestrator for Docker. A quick look at 11 of our servers show that we are running Docker-Compose 2.33 with no issues. 

I will say though Docker-Compose 2 is pretty buggy and prefer the stability of the 1.x python based branch.

**Reply by vmcosco** [reply]
Thanks
